src/overlays/actors/ovl_En_Cow/z_en_cow.c

```diff
@@ -70,32 +71,32 @@ void func_809DEE00(Vec3f* vec, s16 rotY) {
     vec->x = xCalc;
 }
 
-void func_809DEE9C(EnCow* this) {
+void EnCow_PositionCow(EnCow* this) {
```

Comment:
I think MM's `EnCow_SetColliderPos` is better since this function indeed deals with positioning the colliders